deployments/fpga_admissionwebhook/rbac/leader_election_role.yaml

```diff
@@ -0,0 +1,32 @@
+# permissions to do leader election.
+apiVersion: rbac.authorization.k8s.io/v1
```

Comment:
I'm curious, did the leader election work previously?

Reply:
looks like no and it's still broken. I have a fix available soon.